### ISSUE TITLE
VSCode Language Support Extension looks for parser directory that doesn't exist

### ISSUE THREAD

**Issue by luqven** [snowpackjs/astro#373]
## Issue
What I assume is the "official" VSCode language support [extension](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode) for `.astro` errors out immediately after opening a file. 

## Error
So syntax highlighting is all there but then the LSE just craps out because of the bellow error

```console
Error: Cannot find module '/Users/Luis/Documents/experiments/hello-astro/astro-languageserver'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:934:15)
    at Module._load (internal/modules/cjs/loader.js:779:27)
    at Function.f._load (electron/js2c/asar_bundle.js:5:12684)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:72:12)
    at internal/main/run_main_module.js:17:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}
```

## Things I've tried
I tried building all the packages in `./tools/` [directory](https://github.com/snowpackjs/astro/tree/main/tools) and just copy pasting them into my Astro starter but no luck 🤷🏼 

## Behavior I expected
Would be nice to have a language extension for `.astro` files. Wouldn't mind having to build it from source but not entirely sure from the contributing docs how to do that.

Keep up the great work on the framework! I'm excited to finally be giving it a try.

---
Edit: [MatsDahlin](https://github.com/snowpackjs/astro/issues/373#issuecomment-860269983) says rolling "back to previous version of extension worked (0.4.2)". Haven't tried this myself.

**Comment by theZieger** [comment]
Happens to me too. Did a fresh install of Visual Studio Code (on Ubuntu 20.10) just for the Astro language support. But keeps crashing with above error over and over again.

Is there anything we could do to help here from our end?

**Comment by MatsDahlin** [comment]
This happens to me too, and I just watched a Twitch stream where streamer had the same error. 
For me backing to previous version of extension worked (0.4.2), but streamer could not do that because of incompatibility with his VSCode version.

**Comment by FredKSchott** [comment]
@natemoo-re can you take a look?

**Comment by natemoo-re** [comment]
This has been fixed in #467 and released as [`astro-build.astro-vscode@0.5.0`](https://marketplace.visualstudio.com/items?itemName=astro-build.astro-vscode). Please let me know if you run into any more issues!
